libs/design-system-components/src/components/ExtensionEditStep2Form/index.tsx

```diff
@@ -198,4 +192,14 @@ const schema = z.object({
     .optional()
     .transform(e => (e === '' ? undefined : e)),
   nsfw: z.boolean().optional(),
+  links: z.array(
```

Comment:
links are max 10 - according to the model

Reply:
I do the check somewhere else, but I will add it also here